pages/common/yapf.md

```diff
@@ -0,0 +1,16 @@
+# yapf
+
+> Python style guide checker.
+> Source code: https://github.com/google/yapf
```

Comment:
We actually have a special format for links like this:
```suggestion
> More information: <https://github.com/google/yapf>.
```

Reply:
Sorry, it's my first addition to this project =D Noted for the future and I will commit your suggestions

Reply:
No problem whatsoever! 🙂